third-party-drivers/phantomjs.rst

```diff
@@ -0,0 +1,85 @@
+PhantomJSDriver
```

Comment:
Yes, for the moment is small and mostly everything that i have written in this PR is contained in the README.md of the repo.